scripts/common-functions.sh

```diff
@@ -281,7 +281,7 @@ function bench_cmd {
   echo -n " --peer 127.0.0.1:"`get_port $((i-1))`
   echo -n " $(logs "" node_auxx.log)"
   echo -n " --system-start $system_start"
-  echo -n " cmd --commands \"send-to-all-genesis $time $conc $delay $sendmode ./tps-sent.csv\""
+  echo -n " cmd --commands \"send-to-all-genesis $time $time $conc $delay $sendmode ./tps-sent.csv\""
```

Comment:
is `$time` duplicated?

Reply:
ah, I see: the first is "genesis" txs, the second the subsequent txs. 
Then, `$sendmode` is obsolete, I think. It's not part of the structure anymore.